bin/bookkeeper-cli

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env bash
```

Comment:
any reason for keeping it in bookkeeper/bin folder instead of bookkeeper/bookkeeper-server/bin ?

Reply:
the thought here is eventually moving bin/ conf/ directories to the root module. since we will split bookkeeper-server into client and server module in future, and bookkeeper-dist is for packing the binary distribution. moving bin to top-level will make things easier and manageable. I am doing this for this new bookkeeper CLI first and will move other script after we split bookkeeper-server module in 4.8.